ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -50,14 +56,39 @@ const styles = reactStyles({
   title: {
     color: colors.primary,
     fontSize: 16,
-    fontWeight: 600
+    fontWeight: 500,
+    width: '40%',
+    display: 'inline',
+    alignItems: 'flexEnd'
   },
   uneditableProfileElement: {
     paddingLeft: '0.5rem',
     marginRight: 20,
     marginBottom: 20,
     height: '1.5rem',
     color: colors.primary
+  },
+  fadebox: {
+    margin: '1rem 0 0 3%',
```

Comment:
Some of these new styles have unusual % values. How were those calculated?

Reply:
I actually re-used this from workspace page